### ISSUE TITLE
server error 500 after debug=false

### ISSUE THREAD

**Issue by harsh-9in** [harsh-9in/Blogger#221]
**Describe the bug**
The project runs quite fine when debug=True in settings.py but after debug=false it shows server error 500.

**Additional context**
Stackoverflow and other articles are useful for resolving the bug.

**Comment by Amit366** [comment]
I am interested

**Comment by harsh-9in** [comment]
Go ahead @Amit366 and can you plz tell me the program under you are participating.

**Comment by Amit366** [comment]
The program is DWOC

**Comment by Amit366** [comment]
Can you tell me which file should I look into?

**Comment by harsh-9in** [comment]
> Can you tell me which file should I look into?
settings.py looks like creating this problem.

**Comment by Amit366** [comment]
I am getting error that the python version needs to be updated

**Comment by Amit366** [comment]
The error is No name 'config' in module 'decouple'pylint(no-name-in-module)

**Comment by Amit366** [comment]
Could you please help me in this

**Comment by harsh-9in** [comment]
> The error is No name 'config' in module 'decouple'pylint(no-name-in-module)

you need to install requirements so that required packages get istall

**Comment by Amit366** [comment]
I did install the packages but still it is showing this error

**Comment by harsh-9in** [comment]
can you show me the screenshot of error so that it will be easier for Me to solve

**Comment by Amit366** [comment]
Ok I will show it tomorrow

**Comment by Amit366** [comment]
![Uploading IMG_20201228_154223.jpg…]()
This is the error.

**Comment by radhikasingh13** [comment]
@harsh-9in i am interested in this issue.

**Comment by Amit366** [comment]
![IMG_20201228_154223](https://user-images.githubusercontent.com/60662775/103280730-1bf93a00-49f7-11eb-8f22-13a3d0448c4e.jpg)
This is the error.

**Comment by radhikasingh13** [comment]
@harsh-9in i want to know that we need run it locally and then correct it or deploying it on heroku and then run it.Because i find that your project is no problem when debug==false and i commented out these two lines import django_heroku and django_heroku.settings(locals()).So the problem is with heroku .

**Comment by radhikasingh13** [comment]
Do we need to signup on heroku for this.

**Comment by harsh-9in** [comment]
> Do we need to signup on heroku for this.

no you don't need to do this.

**Comment by harsh-9in** [comment]
> @harsh-9in i want to know that we need run it locally and then correct it or deploying it on heroku and then run it.Because i find that your project is no problem when debug==false and i commented out these two lines import django_heroku and django_heroku.settings(locals()).So the problem is with heroku .

are you able to run the project locally when debug=false?

**Comment by radhikasingh13** [comment]
yes i am able to run the project locally even when debug==false when i commented out these two lines from setting.py file
import django_heroku and django_heroku.settings(locals()

**Comment by harsh-9in** [comment]
okk fine can you create a pr so i can review the code.

**Comment by radhikasingh13** [comment]
@harsh-9in ,I have made the PR. Please check it out.
